test/clean-testdir

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# This file is part of Cockpit.
+#
+# Copyright (C) 2015 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+SELF=delete-errorfiles
+
+CLEAN_PACKAGES=0
+CLEAN_TESTFILES=0
+CLEAN_VM=0
+CLEAN_ALL=0
+DRY_RUN="-f"
+
+usage()
+{
+    echo >&2 "Usage: $SELF [-h] [-d] [-a] [-p] [-t] [-r]"
+    echo >&2 "        -h Display info on usage"
+    echo >&2 ""
+    echo >&2 "        -d Dry run: don't actually clean anything"
+    echo >&2 ""
+    echo >&2 "        -a Clean directory of all non-versioned files"
+    echo >&2 ""
+    echo >&2 "        -p Delete created packages (*.rpm)"
+    echo >&2 "        -t Delete test files (attachments, screenshots, journals, logs)"
+    echo >&2 "        -r Run vm-reset"
+}
+
+while getopts "hdaptr" OPTION
+do
+    case $OPTION in
+        h)
+            usage
+            exit 0
+            ;;
+        d)
+            DRY_RUN="-n"
+            ;;
+        a)
+            CLEAN_ALL=1
+            ;;
+        p)
+            CLEAN_PACKAGES=1
+            ;;
+        t)
+            CLEAN_TESTFILES=1
+            ;;
+        r)
+            CLEAN_VM=1
+            ;;
+     esac
+done
+
+if [ $CLEAN_ALL == 1 ]; then
+    echo >&2 "Cleaning everything"
+    git clean "$DRY_RUN" -d -x
```

Comment:
Should we use ```git -C $BASE``` where ```BASE=$(dirname $0)``` ? This script currently makes assumptions about the directory it is run in.